static/src/js/providers/AuthContextProvider/AuthContextProvider.jsx

```diff
@@ -109,6 +146,7 @@ const AuthContextProvider = ({ children }) => {
     const interval = setInterval(async () => {
       checkAndRefreshToken(loginInfo, handleRefreshToken)
     }, 1000)
+    // TODO do we really need to check the token every second?
```

Comment:
No, in fact in recent testing, it was causing issues where it would fetch multiple refresh cookies.  I modified this to use 5 seconds and that seem to fix, so this is one change I was planning on doing.

Reply:
The fact that we have a setInterval at all is super sus to me.